eng/scripts/Validate-TypeSpec.ps1

```diff
@@ -4,30 +4,42 @@ param (
   [switch]$GitClean = $false
 )
 
-$exitCode = 0
+. $PSScriptRoot/Logging-Functions.ps1
 
 $typespecFolders = &"$PSScriptRoot/Get-TypeSpec-Folders.ps1" -CheckAll:$CheckAll
 
-Write-Host "typespecFolders:"
-foreach ($typespecFolder in $typespecFolders) {
-  Write-Host "  $typespecFolder"
-}
-Write-Host
-
+$typespecFoldersWithFailures = @()
 if ($typespecFolders) {
   $typespecFolders = $typespecFolders.Split('',[System.StringSplitOptions]::RemoveEmptyEntries)
   foreach ($typespecFolder in $typespecFolders) {
+    LogGroupStart "Validating $typespecFolder"
+    LogInfo "npx --no tsv $typespecFolder"
     npx --no tsv $typespecFolder 2>&1 | Write-Host
     if ($LASTEXITCODE) {
-      $exitCode = 1
+      $typespecFoldersWithFailures += $typespecFolder
+      $errorString = "TypeSpec Validation failed for project $typespecFolder run the following command locally to validate."
+      $errorString += "`n > npm install"
```

Comment:
Same comment I made for fix instructions in `ci-fix.md`

Reply:
I'll change this to npm ci but I'm sure we should add all those git steps in this message. My hope is that this message can be pretty much copy and paste for folks.